activity_otp_verify.xml

```diff
@@ -20,6 +20,8 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
             app:srcCompat="@drawable/ic_otp" />
+        
+        this the typing session i have done so far by the best  which i could describe to you

```

Comment:
hey do this in a nice way so that the user could understand the code easily

Reply:
thanks for your feedback i would make the necessary changes & commit again

Reply:
### Again need to ask about the typing session

**How's it going so far?**_I think you would do great_

> You could alternatively prefer doing by yourself
`sudo apt-get install upadate`

- tlelr
- lsdkjfsla
- dslfjasdl
- @ayasaack